pom.xml

```diff
@@ -13,18 +12,6 @@
   <packaging>hpi</packaging>
   <name>Workspace Rotation Plugin</name>
   <url>http://wiki.hudson-ci.org/display/HUDSON/Workspace+Rotation+Plugin</url>  
-  <build>
```

Comment:
@jglick If you are updating Maven configuration, it would be interesting to update this URL.